### DISCUSSION TITLE
Separate the list of artist's albums into "the artist's albums" and "albums the artist is featured on"

### DISCUSSION THREAD

**Discussion by SteveDinn** [navidrome/navidrome#3400] [Ideas]
Currently, when viewing the artist page, all allbums that the artist has any songs on are featured in one chronological list.

I would love it if it could be configured such that there were two lists
- one containing the artist's albums, i.e., albums where artist == album artist.
- a second list containing the albums on which the artist is featured, i.e., artist != album artist.

In this way, the compliaitions, or albums with tracks featuring an artist could be separated from the artists actual albums.

Thoughts?

**Comment by SteveDinn** [comment]
Currently, when you go to queue up all albums by a particular artist, you may end up getting tracks that have nothing to do with said artist, but just happen to exist on an album where the artist is featured, and this may not be the intent.

**Comment by deluan** [comment]
Yeah, we will work on a better artist page in the new UI. This is similar to how Airsonic-Refix, Feishin and others groups the albums, and we will do the same in the new UI.

**Reply by dweymouth** [reply]
Do you know how Airsonic-Refix manages to group albums like this? when the Subsonic api doesn't provide a way to differentiate them (or even to query the "appears on" albums by artist ID?)

**Reply by deluan** [reply]
Ops, my bad. Only Feishin does this (using ND's API). 
Maybe you can check if the album's `artistId` is the same as the `artistId` of the current artist (as returned by `getArtist` endpoint)?

**Comment by metalheim** [comment]
Linking to https://github.com/navidrome/navidrome/issues/961 as it is very much related and already describes this functionality :)
